### PR TITLE
Security update: Update Node.js dependency from v6 to v8

### PR DESCRIPTION
**Pull request by oguzcankirmemis**
Should only be merged after [#68](https://github.com/Open-IoT-Service-Platform/oisp-frontend/pull/68) this commit, or it will break since the Jsonwebtoken module wants that the JWT Token is in timestamp format.

Changes that are done:

Updates to package.json: Almost all modules updated so that no security vulnerability remains

Updates to Winston logger: Because of the module update, the format of the config file for winston has changed

Updates to Dockerfile: Because there are some modules that is not prebuilt in alpine, some dev tools such as (python, g++, .gyp) is required, these are added and then deleted after the installation

### CODE REVIEW

**Comment by Cloud-Chen** [comment]
Can one of the admins verify this patch?

**Review by arkocal — CHANGES_REQUESTED** [review]
Please comment the apk del line in Dockerfile, it looks a little bit counter-intuitive that they are required for npm install.

**Review by arkocal — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,15 @@
-FROM node:6-alpine
+FROM node:8-alpine
 
 RUN apk update; apk add ncurses make bash
```
Comment (arkocal) [inline Dockerfile]:
```suggestion
RUN apk update; apk add --no-cache --virtual ncurses make bash .gyp python g++
```
Is there anything that prevents us from doing this? (obviously, you will also have to remove the followin line)
In general, it is better to squash docker commands that belong to the same logical task, as this creates less layers.

Comment (oguzcankirmemis) [reply Dockerfile]: You are right, it is now as you suggest it. I have also done the same change at the frontend now.

**Pull request merged**